examples/keras_rs/deep_recommender.py

```diff
@@ -429,9 +429,7 @@ def preprocess_rating(x):
             "raw_user_age": features["raw_user_age"],
             "user_gender": features["user_gender"],
             "user_occupation_label": features["user_occupation_label"],
-            "user_gender_X_raw_user_age": tf.squeeze(
-                features["user_gender_X_raw_user_age"], axis=-1
-            ),
+            "user_gender_X_raw_user_age": features["user_gender_X_raw_user_age"],
```

Comment:
Adding `tf.squeeze` is the fix I'd pushed to make the example work on Colab. Curious how removing it made it work for you